### PR TITLE
Doc: Document prefetching behavior

### LINKED ISSUE #210: Document prefetching behavior and benefits

### PR DESCRIPTION
**Pull request by nicksay**
Closes #210

### CODE REVIEW

File: doc/documentation/prefetching.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+|:------------|:------------------------------|:-------|
+| `onRequest` | Started; Sending Prefetch     | Abort  |
+| `onProcess` | Processing; Response Received | Abort  |
+| `onDone`    | Done                          |        |
+
+When SPF sends the prefetch request to the server, it will
+append the configurable identifier to the URL in the same manner
+as navigation; by default, this value will be `?spf=prefetch`.
+
+When the prefetched response has been
```
Comment (awbraunstein) [inline doc/documentation/prefetching.md]: super nit: can you re-fill these lines so they are more even? If that isn't something your editor can do very easily don't worry about it.

File: doc/documentation/prefetching.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+received, SPF will place it in the local cache as eligible for
+one "new" navigation.  After that one use, the cached response
+will only be eligible for "history" navigation.  However, if the
+`cache-unified` config value is set to `true`, then the
+prefetched response will be available for all navigation like
```
Comment (awbraunstein) [inline doc/documentation/prefetching.md]: s/navigation/navigations/

File: doc/documentation/prefetching.md (hunk 1 above)
Comment (nicksay) [inline doc/documentation/prefetching.md]: Done

File: doc/documentation/prefetching.md (hunk 2 above)
Comment (nicksay) [inline doc/documentation/prefetching.md]: Done.

**Comment by awbraunstein** [comment]
LGTM :dolphin:

**Pull request merged**